src/components/Strawpoll.tsx

```diff
@@ -82,12 +88,45 @@ function getNumberOfVotes(option: StrawpollOptionData, medium: StrawpollMedium)
   }
 }
 
+export function StrawpollConfirm(props: ModalProps) {
+  const onYesClick = () => {
+    props.onConfirm()
+    props.setOpen(false)
```

Comment:
I would personally call this `onChangeOpenState` to use the `on` prefix convention for callbacks